docs/kr/core-concepts/building-blocks/tasks.md

```
@@ -0,0 +1,123 @@
+
+  **Key parameters:**
+
+  * `executable`: Any Python callable (function, coroutine, generator, async generator)
+  * `task_config`: Configuration for batching, error handling, and logging
+  * `default_params`: Parameters that are always passed to the executable
+</Accordion>
+
+<Accordion title="Supported Task Types">
+  Cognee automatically detects and handles different Python function types:
+
+  * **Functions**: Standard synchronous functions
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)